app-arch/tar/tar-1.29.recipe

```diff
@@ -26,17 +26,33 @@ PROVIDES="
 REQUIRES="
 	haiku
 	"
+
 BUILD_REQUIRES="
 	haiku_devel
 	"
 BUILD_PREREQUIRES="
 	cmd:automake
 	cmd:autoreconf
+	cmd:find
 	cmd:gcc
 	cmd:ld
 	cmd:make
 	cmd:sed
 	"
+if [ "$recipeAction" = TEST ]; then
+	BUILD_PREREQUIRES="$BUILD_PREREQUIRES
```

Comment:
I also though about having the exact same TEST_PREREQUIRES ;-)
There would be, however, a small question to decide: should haikuporter consider that any item in BUILD_PREREQUIRES should be repeated in TEST_PREREQUIRES if it is also needed during the TEST() ?
If we say that items should be repeated, then it might be worth using something like:
```
PREREQUIRES="
	cmd:find
	cmd:make
	cmd:sed
	"
BUILD_PREREQUIRES="$PREREQUIRES
	cmd:automake
	cmd:autoreconf
	cmd:gcc
	cmd:ld
	"
TEST_PREREQUIRES="$PREREQUIRES
	cmd:compress
	cmd:gzip
	cmd:bzip2
	"
```

But this is a big change. Maybe we should think about it for while before we change anything.
Hmm, I think we also need gcc for the tests, so maybe the best solution is to simply append items to BUILD_PREREQUIRES if "$recipeAction" = TEST, like I did for the recipe. I guess this would be the first recipe to have such a conditional BUILD_PREREQUIRES. I think it's a good idea to keep that variable clean from test prerequisites, even if it introduces a small change in the layout of recipes that have a TEST. The nice thing is that it does not break anything and that we can "migrate" smoothly to the new layout everytime a recipe needs an update.